app/src/Components/Game.js

```diff
@@ -0,0 +1,124 @@
+import * as React from "react";
+
+import * as apiClient from "../apiClient";
+import { calculateWinner } from "../helpers";
+
+import AddScore from "./AddScore";
+import Board from "./Board";
+import ScoreBoard from "./ScoreBoard";
+import "./Game.scss";
+
+const Game = () => {
+  const {
+    players,
+    loadPlayers,
```

Comment:
This seems like a weird way to include all the API calls into this component. Was there a reason you needed to wrap the api calls in a custom hook like this? Seems like it would have been easier to just important all the methods from ApiClient directly.